docs/docs/reference/02-manual-installation-launch.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 1
+title: Manual Installation and Launch
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The following sections explain how the installation and launch steps of the AICA Launcher can be performed manually.
+The pre-requisites are still a valid license and a host with Docker installed. For the rest of this guide, it will be
+assumed that a valid license has been saved to a file called `aica-license.toml` on the host machine.
```

Comment:
```suggestion
The following sections explain how to install and launch AICA Core and any additional packages manually from the command
line without the use of AICA Launcher. The pre-requisites are still a valid license and a host with Docker installed.
For the rest of this guide, it will be assumed that a valid license has
been saved to a file called
`aica-license.toml` on the host machine.
```